plugins/woocommerce/src/Blocks/Domain/Bootstrap.php

```diff
@@ -427,6 +424,15 @@ function () {
 				return new QueryFilters();
 			}
 		);
+
+		if ( wc_current_theme_is_fse_theme() ) {
```

Comment:
Same here. :slightly_smiling_face: 

```suggestion
		if ( wc_current_theme_is_fse_theme() || current_theme_supports( 'block-template-parts' ) ) {
```